services/src/test/scala/cromwell/services/womtool/impl/WomtoolServiceInCromwellActorSpec.scala

```diff
@@ -0,0 +1,150 @@
+package cromwell.services.womtool.impl
+
+import akka.pattern._
+import akka.testkit.TestProbe
+import com.typesafe.config.ConfigFactory
+import cromwell.core.WorkflowSourceFilesCollection
+import cromwell.languages.config.{CromwellLanguages, LanguageConfiguration}
+import cromwell.services.ServicesSpec
+import cromwell.services.womtool.WomtoolServiceMessages._
+import wom.core._
+
+class WomtoolServiceInCromwellActorSpec extends ServicesSpec("Womtool") {
+
+  val womtoolActor = system.actorOf(WomtoolServiceInCromwellActor.props(ConfigFactory.empty(), ConfigFactory.empty(), TestProbe().ref))
+  CromwellLanguages.initLanguages(LanguageConfiguration.AllLanguageEntries)
+
+  object TestData {
+    val workflowUrlValid = "https://raw.githubusercontent.com/broadinstitute/cromwell/develop/womtool/src/test/resources/validate/wdl_draft3/valid/callable_imports/my_workflow.wdl"
+    val workflowUrlNotFound = "https://raw.githubusercontent.com/broadinstitute/cromwell/develop/my_workflow"
+    val workflowUrlBadHost = "https://zardoz.zardoz"
```

Comment:
ugh you made me Google that... 😱

Reply:
Some might say Sean Connery's finest hour

Reply:
You mean Option might say...

Reply:
You can thank my CS program's functional programming class, e.g. https://www.cs.cornell.edu/courses/cs3110/2013sp/hw/ps1/ps1.html

I believe there is an annual Zardoz screening after the final, though I did not go